web/vtctld2/src/app/dashboard/shard.component.ts

```diff
@@ -97,15 +101,19 @@ export class ShardComponent implements OnInit, OnDestroy {
         {label: 'Run Health Check', command: (event) => {this.openRunHealthCheckDialog(); }},
       ]},
       {label: 'Change', items: [
-        {label: 'Set ReadOnly', command: (event) => {this.openSetReadOnlyDialog(); }},
-        {label: 'Set ReadWrite', command: (event) => {this.openSetReadWriteDialog(); }},
+        {label: 'Ignore Health Error', command: (event) => {this.openIgnoreHealthErrorDialog(); }},
         {label: 'Start Slave', command: (event) => {this.openStartSlaveDialog(); }},
         {label: 'Stop Slave', command: (event) => {this.openStopSlaveDialog(); }},
-        {label: 'Ignore Health Error', command: (event) => {this.openIgnoreHealthErrorDialog(); }},
-        {label: 'Delete', command: (event) => {this.openDeleteTabletDialog(); }},
+        {label: 'Delete Tablet', command: (event) => {this.openDeleteTabletDialog(); }},
       ]},
     ];
     if (this.featuresService.activeReparents) {
+      // Add a couple new items to the 'Change' section.
+      result[1].items.push(
+        {label: 'Set ReadOnly', command: (event) => {this.openSetReadOnlyDialog(); }},
+        {label: 'Set ReadWrite', command: (event) => {this.openSetReadWriteDialog(); }},
+      );
+      // Add an entire section.
       result.push(
```

Comment:
This looks confusing to me.

You're pushing the "Reparent" section for a second time here? (line 86 and line 118)

Am I missing something?

I would expect that some of these menu entries are conditional on the tablet type?